src/pybind/mgr/rbd_support/schedule.py

```diff
@@ -319,9 +321,23 @@ def remove(self,
                start_time: Optional[StartTime] = None) -> None:
         self.items.discard((interval, start_time))
 
+    def find(self,
+             interval: Interval,
+             start_time: Optional[StartTime] = None) -> bool:
```

Comment:
I can't understand right now if we really need this `find` function. But it does not look like "a pythonic way" to me.

Reply:
This is to validate we are removing from existing set of schedules.
With this patch:
sunny@o05 build -> rbd mirror snapshot schedule ls --pool data --cluster site-1 --recursive
POOL  NAMESPACE  IMAGE   SCHEDULE
data             image1  every 5m
sunny@o05 build -> rbd --cluster site-1 mirror snapshot schedule remove 1m --pool data --image image1
rbd: rbd mirror snapshot schedule remove failed: (22) Invalid argument: Invalid schedule

without:
sunny@o05 build -> rbd mirror snapshot schedule ls --pool data --cluster site-a --recursive
POOL  NAMESPACE  IMAGE   SCHEDULE
data             image1  every 5m
sunny@o05 build -> rbd --cluster site-a mirror snapshot schedule remove 1m --pool data --image image1

Reply:
Do you mean that when removing non-existent schedule it just returns with succes (but nothing bad happens)? Actually I am not sure it is a bug and we need an improvement here. Anyway I think it should be a separate PR.

Reply:
Created tracker for this; will address as a separate PR : https://tracker.ceph.com/issues/54129

Reply:
Created tracker for this (https://tracker.ceph.com/issues/54129); will raise another PR for that.